tests/test_users.py

```diff
@@ -75,3 +78,24 @@ def test_now_in_my_timezone(self):
         import datetime
 
         self.assertIsInstance(now, datetime.datetime)
+
+
+@pytest.mark.skipif(
+    "os.getenv('TRAVIS', False) or os.getenv('GITHUB_ACTIONS', False)",
+    reason="Skipping tests when using Travis or Github Actions, as not we cannot delete new user",
+)
+class TestCreateUsers(TestBetamax):
+    def test_create_a_new_user(self):
```

Comment:
Can you add a test for the `reset_password()`? To keep the good coverage going.

Reply:
sure!